templates/test/list_table.html

```diff
@@ -19,7 +19,7 @@
                         <td data-title="Run Errors">{{ "Unknown" if not test.finished else ("Yes" if test.failed else "No") }}</td>
                         <td data-title="Completed">
                             <a href="{{ url_for('test.by_id', test_id=test.id) }}">{{ "Yes" if test.finished else "No" }}</a>
-                            {% if user.is_admin or "{{ url_for(request.endpoint) }}" == "{{ url_for('custom.index'}}" %}
+                            {% if user.is_admin or url_for(request.endpoint) == url_for('custom.index') %}
```

Comment:
I'm still wondering why you compare using url_for when you can just compare the endpoints?